api/src/main/java/keywhiz/api/automation/v2/ClientDetailResponseV2.java

```diff
@@ -0,0 +1,42 @@
+package keywhiz.api.automation.v2;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.auto.value.AutoValue;
+import java.time.OffsetDateTime;
+import keywhiz.api.model.Client;
+
+@AutoValue public abstract class ClientDetailResponseV2 {
+  ClientDetailResponseV2() {} // prevent sub-classing
+
+  public static ClientDetailResponseV2 fromClient(Client client) {
+    return new AutoValue_ClientDetailResponseV2(
+        client.getName(),
+        client.getDescription(),
+        client.getCreatedAt(),
+        client.getUpdatedAt(),
+        client.getCreatedBy(),
+        client.getUpdatedBy());
+  }
+
+  /**
+   * Static factory method used by Jackson for deserialization
+   */
+  @SuppressWarnings("unused")
+  @JsonCreator public static ClientDetailResponseV2 fromParts(
+      @JsonProperty("name") String name,
+      @JsonProperty("description") String description,
+      @JsonProperty("creationDate") OffsetDateTime creationDate,
```

Comment:
I'm still planning on adding the integer timestamp, but I think I'll put it in the next PR to avoid blocking things.